### PR TITLE
Bump vale version

### PR DESCRIPTION
**Pull request by SMoraisAnsys**
~~Test new vale version which seems to be solving recent problems:~~
~~- [ ] https://github.com/ansys/actions/issues/350 (single back ticks)~~
~~- [ ] https://github.com/errata-ai/vale/issues/706 (titles)~~

~~If everything is working correctly, the ansys action should be updated with version 2.29.5~~

The vale update triggers new errors that should be reported. Do not bother with this PR as I'll only use it to die and retry vale updates.

---

**Updates**: seems like following https://github.com/errata-ai/vale/issues/706 tips (end of conversation) did the trick for many issues related to `vale` / `reviewdog`. See file changes in https://github.com/ansys/pyansys-dev-guide/pull/371/files#diff-0921d6da005aab688da52d333dc4db8e897fbe1e5f9083c5b4ab60381af78773

### CODE REVIEW

**Comment by SMoraisAnsys** [comment]
Single back ticks is still present :/

**Comment by SMoraisAnsys** [comment]
I'll rework the history of this PR but I think it should be integrated soon :)

File: doc/styles/Vocab/ANSYS/accept.txt [hunk 1]
```diff
@@ -1,100 +1,94 @@
...
+[Hh]otfixes
 Hpertext
+(?i)Html
 initialisms
 initializer
 internetworking
-Isort
-isort
+[Ii]sort
+Kaszynski
```
Comment (MaxJPRey) [inline doc/styles/Vocab/ANSYS/accept.txt:44]:
```suggestion
```

**Review by RobPasMue — APPROVED** [review]

File: doc/source/conf.py [hunk 2]
```diff
@@ -32,6 +32,7 @@
 
 html_theme_options = {
     "github_url": "https://github.com/ansys/dev-guide",
+    "navigation_with_keys": False,
```
Comment (RobPasMue) [inline doc/source/conf.py:35]: Not needed anymore with the new Ansys sphinx theme.
```suggestion
```

**Comment by RobPasMue** [comment]
What's the status here @SMoraisAnsys ¿?

**Comment by RobPasMue** [comment]
Thanks @SMoraisAnsys!

**Pull request merged**